docs/csharp/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md

```diff
@@ -1,379 +1,371 @@
+https://docs.microsoft.com                                       39,639
+https://docs.microsoft.com/azure                                401,180
+https://docs.microsoft.com/powershell                            58,240
+https://docs.microsoft.com/dotnet                                69,693
+https://docs.microsoft.com/aspnet/core                           87,771
+https://docs.microsoft.com/windows                               25,580
+
+Total bytes returned:  682,103
+Elapsed time:          00:00:06.1445331
+
+Control returned to OnStartButtonClick.
 ```
 
 Notice that it takes a few seconds to display the counts. During that time, the UI thread is blocked while it waits for requested resources to download. As a result, you can't move, maximize, minimize, or even close the display window after you choose the  **Start** button. These efforts fail until the byte counts start to appear. If a website isn't responding, you have no indication of which site failed. It is difficult even to stop waiting and close the program.
 
 ## Convert GetURLContents to an asynchronous method
 
-1. To convert the synchronous solution to an asynchronous solution, the best place to start is in `GetURLContents` because the calls to the <xref:System.Net.HttpWebRequest> method <xref:System.Net.HttpWebRequest.GetResponse%2A> and to the <xref:System.IO.Stream> method <xref:System.IO.Stream.CopyTo%2A> are where the application accesses the web. .NET Framework makes the conversion easy by supplying asynchronous versions of both methods.
+1. To convert the synchronous solution to an asynchronous solution, the best place to start is in `ProcessUrl` because the calls to the <xref:System.Net.HttpWebRequest> method <xref:System.Net.HttpWebRequest.GetResponse%2A> and to the <xref:System.IO.Stream> method <xref:System.IO.Stream.CopyTo%2A> are where the application accesses the web. The .NET runtime makes the conversion easy by supplying asynchronous versions of both methods.
 
-     For more information about the methods that are used in `GetURLContents`, see <xref:System.Net.WebRequest>.
+     For more information about the methods that are used in `ProcessUrl`, see <xref:System.Net.WebRequest>.
 
     > [!NOTE]
-    > As you follow the steps in this walkthrough, several compiler errors appear. You can ignore them and continue with the walkthrough.
+    > As you follow the steps in this tutorial, several compiler errors appear. You can ignore them and continue with the tutorial.
 
-     Change the method that's called in the third line of `GetURLContents` from `GetResponse` to the asynchronous, task-based <xref:System.Net.WebRequest.GetResponseAsync%2A> method.
+     Change the method that's called in `ProcessUrl` from `GetResponse` to the asynchronous, task-based <xref:System.Net.WebRequest.GetResponseAsync%2A> method.
 
     ```csharp
-    using (WebResponse response = webReq.GetResponseAsync())
+    using WebResponse response = webReq.GetResponseAsync();
```

Comment:
Very nice to see the new C# features are used more and more in docs. I've seen developers that write new C# code that looks 10-year-old 😄.